arch/arm/src/imxrt/imxrt_serial.c

```diff
@@ -2629,7 +2629,7 @@ static int imxrt_ioctl(struct file *filep, int cmd, unsigned long arg)
 
         if ((arg & SER_SINGLEWIRE_ENABLED) != 0)
           {
-            uint32_t gpio_val = IOMUX_OPENDRAIN;
+            uint32_t gpio_val = 0;
```

Comment:
Why re-move the information?

Reply:
Maybe it is appropriate for the IMX, but we have seen multiple drivers on the line and thresholding issues so the control was added for the PU control.

But in any case use IOMUX_PUSHPULL so there is not loss of information.